models/token.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var Token = sequelize.define('Token', {
+    accessToken: {
+      type: DataTypes.STRING,
+    },
+    refreshToken: {
+      type: DataTypes.STRING,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+    },
+    userAgent: {
+      type: DataTypes.STRING,
+    },
+  }, {
+      classMethods: {
```

Comment:
class method sepertinya sudah tidak bisa dipakai lagi di versi 4.

berikut pemakaian sesuai docs:

http://docs.sequelizejs.com/manual/tutorial/upgrade-to-v4.html

**old**
```
const Model = sequelize.define('Model', {
    ...
}, {
    classMethods: {
        associate: function (model) {...}
    },
    instanceMethods: {
        someMethod: function () { ...}
    }
});
```

**new**
```
const Model = sequelize.define('Model', {
    ...
});

// Class Method
Model.associate = function (models) {
    ...associate the models
};

// Instance Method
Model.prototype.someMethod = function () {..}
```